.circleci/config.yml

```diff
@@ -102,23 +102,33 @@ jobs:
               if [[ -f "$subdir"/test_requirements.txt ]]; then
                 pip install -r "$subdir"/test_requirements.txt
               fi
+              if [[ -f "$subdir"/requirements_dev.txt ]]; then
+                pip install -r "$subdir"/requirements_dev.txt
+              fi
```

Comment:
Are you introducing a new convention here?

All* Ocean packages have one `requirements.txt` in root and then one in `tests/` and possibly `docs/`. Your `requirements_dev.txt` would be `tests/requirements.txt` in our current convention.

*) all except minorminer. Not sure why we deviated with `test_requirements.txt` there. :thinking:

Reply:
I guess I am, although not intentionally. This was never questioned previously, but I'd be happy to follow conventions as much as possible. Can switch to `tests/requirements.txt` if that's the preferred one.